lib/rules/selector-max-class/__tests__/index.js

```diff
@@ -0,0 +1,130 @@
+"use strict"
+
+const messages = require("..").messages
+const ruleName = require("..").ruleName
+const rules = require("../../../rules")
+
+const rule = rules[ruleName]
+
+testRule(rule, {
+  ruleName,
+  config: [2],
```

Comment:
reoder test first with config `0`, when `1`, when `2` and last `less`, `scss` and etc